ddl/db_integration_test.go

```diff
@@ -16,6 +16,7 @@ package ddl_test
 import (
 	"context"
 	"fmt"
+	"math"
```

Comment:
Wary about `ALTER TABLE `t` /*T![force_auto_inc] FORCE */ AUTO_INCREMENT = 3;` this syntax may lead the compatible problem between heterogeneous or edtional upstream and downstream. 

This doesn't take effect in MySQL, so implicit insertion in MySQL may take the wrong way out. 
This doesn't take effect in old edition, so implicit insertion in Old TiDB may take the wrong way out.

Reply:
Do you think binlog/ticdc should ignore these FORCE alter statements?